resources/js/components/attribute/Integer.vue

```diff
@@ -84,11 +88,19 @@
                 // only emit @change event if field is validated (required because Entity.vue components)
                 // trigger this watcher several times even if another component is updated/validated
                 if(!v.meta.validated) return;
-                context.emit('change', {
-                    dirty: v.meta.dirty,
-                    valid: v.meta.valid,
-                    value: v.value,
-                });
+                // Apparently, invalid values are returned as empty value in HTML number inputs
+                // Because it is not a required field, it returns true, because an empty value is a valid int
+                // see https://github.com/logaretm/vee-validate/issues/1667#issuecomment-431992508
+                if(!validateValidation(v.value)) return;
+                // also call validate() to make sure all values are up to date
+                // without this, meta/errors still contain the value from last validation
+                validate().then(validation => {
+                    context.emit('change', {
+                        dirty: v.meta.dirty,
+                        valid: v.meta.valid,
+                        value: v.value,
+                    });
+                })
```

Comment:
The value is correctly validated. But:

1. The value being invalid does not prevent the user from submitting => User inserts invalid value and has no indicator to see her mistake => submits (is not prevented) => sees data as if it was updated successfully => data infact remaind the same.
2. Should we just block users from entering other values than numbers (and leading - ?)